docs/node-operators/requirements/data-node-infrastructure.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 2
+title: Data node infrastructure
+sidebar_label: Data node infrastructure
+vega_network: TESTNET
+hide_title: false
+---
+import NetworkParameter from '@site/src/components/NetworkParameter';
+import Topic from '/docs/topics/_topic-data-nodes.mdx';
+
+<Topic />
+
+This page covers the minimum and recommended hardware requirements, and the software requirements for running a data node connecting to the Vega network.
+
+The best hardware may vary depending on how you set up your node and database service, and how you intend to use the data node.
+
+Data node operators who are running a data node to support their own needs can run a leaner setup depending on their own requirements.
+
+Those data node operators running archive nodes, which retain all chain information, will need a more robust setup.
+
+## Hardware recommendations
+
+### System requirements
+| Resource    | Minimum     | Recommended |
+| ----------- | ----------- | ----------- |
+| CPU | 6 cores at >= 3Ghz | 12 cores at >= 4Ghz|
+| RAM   | 16GB            | 32GB        |
+| Storage   | 1TB NVMe SSD| 2TB NVMe SSD |
+
+The impact of using fewer cores than recommended is that the critical parts (data node and non-validators) will be starved of CPU resources, which will reduce the maximum throughput of the network.
+
+The impact of having slower cores than recommended (or older cores that have a reduced IPC rate compared to modern Zen3 cores) is that the maximum throughput will be reduced due to the bottlenecks being single threaded.
```

Comment:
**question (non-blocking)**: And what's the consequence of this?

```suggestion
The impact of having slower cores than recommended (or older cores that have a reduced IPC rate compared to modern Zen3 cores) is that the maximum throughput will be reduced due to the bottlenecks being single threaded. This could cause your validator to fall behind the others, which could lead to your removal from the validator set.
```

Reply:
This is the data node page, so this isn't a factor here.